Forte.EpiServer.AzureSearch/ContentExtractor/Page/PageContentExtractorController.cs

```diff
@@ -1,14 +1,16 @@
 using System.Collections.Generic;
 using System.Linq;
 using EPiServer.Core;
+using Forte.EpiServer.AzureSearch.Extensions;
+using Forte.EpiServer.AzureSearch.Model;
 
-namespace Forte.EpiServer.AzureSearch.Model
+namespace Forte.EpiServer.AzureSearch.ContentExtractor.Page
 {
-    public class ContentExtractorController : IContentExtractorController
+    public class PageContentExtractorController : IPageContentExtractorController
     {
-        private readonly IEnumerable<IContentExtractor> _extractors;
+        private readonly IEnumerable<IPageContentExtractor> _extractors;
 
-        public ContentExtractorController(IEnumerable<IContentExtractor> extractors)
```

Comment:
This is backward compatibility break?

Reply:
I mean I had (in my project) registered custom implementation of IContentExtractor and it was used to extract content. It will be no longer, correct?

Reply:
This is true, now there will be both IPageContentExtractor, IBlockContentExtractor and mixed ones, like IIndexableContentExtractor.

There is an option of leaving the old interface for all extractor and move the decision to the CanExtract method, but I think it's clearer that way.

Reply:
true dat, I see in IContentRepository there's no such notice, there is one in ConterRepository class, which is a bit strange, but still. Nevermind.